### PR TITLE
Fix test_inflate_ip__larger.

### LINKED ISSUE #3096: Incorrect tests & missing test in ``rect_test``
I'm opening this issue as I don't want to work on it for the moment.

I have noticed many errors when I tried to implement ``rel_center`` :  
- ``test_inflate_ip__larger`` does not check what it is supposed to check (duplicate code of ``test_inflate_ip__smaller``)
- ``test_inflate_ip__smaller``'s docstring is incorrect
- each attributes are not checked with literal values if they have the right values. You might think calculating it based on other attributes garantie that the attributes return the right value but this shouldn't be the case ! Not for tests at least ! This is why I would like to see a test that checks if the attributes return the right value based on literal values.

### PR DESCRIPTION
**Pull request by avpai-dinosaur**
This test case was not checking what it was supposed to check (duplicate code of test_inflate_ip__smaller).

Fixes #3096

### CODE REVIEW

File: test/rect_test.py [hunk 1]
```diff
@@ -801,18 +801,18 @@ def test_inflate_ip__larger(self):
...
+        self.assertEqual(r.left - 2, r2.left)
+        self.assertEqual(r.top - 3, r2.top)
+        self.assertEqual(r.right + 2, r2.right)
+        self.assertEqual(r.bottom + 3, r2.bottom)
+        self.assertEqual(r.width + 4, r2.width)
+        self.assertEqual(r.height + 6, r2.height)
 
     def test_inflate_ip__smaller(self):
-        """The inflate method inflates around the center of the rectangle"""
+        """The inflate_ip method inflates around the center of the rectangle"""
```
Comment (bilhox) [inline test/rect_test.py:815]: While you are editing the docstring, it's better if you actually say what does the test do brievly. Same for ``test_inflate_ip__larger``

**Review by bilhox — CHANGES_REQUESTED** [review]
LGTM, thanks for the contribution ! I left a comment for you.

File: test/rect_test.py (hunk 1 above)
Comment (avpai-dinosaur) [reply test/rect_test.py]: Took a stab at a more descriptive docstring, lmk what you think

**Review by bilhox — APPROVED** [review]
LGTM 🥳

**Review by gresm — APPROVED** [review]
LGTM. The docstrings for tests of ``pygame.Rect.scale_by`` could also benefit from updating them, but it's probably out of scope of this PR.

**Comment by bilhox** [comment]
> LGTM. The docstrings for tests of `pygame.Rect.scale_by` could also benefit from updating them, but it's probably out of scope of this PR.

Indeed I can see that for certain tests. I think it's something a bit linked to the topic of this PR. So I would suggest a change there as gresm mentionned it. Otherwise it will not be fixed in a near future.

**Review by MyreMylar — APPROVED** [review]
LGTM 👍

**Pull request merged**